### ISSUE TITLE
S2VTAttModel doesn't converge

### ISSUE THREAD

**Issue by dmandreev** [xiadingZ/video-caption-pytorch#9]
Hello DingXia! I'm trying to repro your results and train S2VTAttModel  with train data you linked in read.me with train.py params  --gpu 0,1,2,3 --epochs 9001 --batch_size 450 --checkpoint_path data/save --feats_dir data/feats/resnet152/train-video --dim_vid 2048 --model S2VTAttModel
But after 4 days of training, train/loss still too high.  iter 12 (epoch 5463), train_loss = 23.042719 Does I'm doing something wrong?

**Comment by xiadingZ** [comment]
It's normal, my model also has this train_loss. you can test it to see blue/cider... score
